### PR TITLE
chore: setup page with some examples

### PR DESCRIPTION
**Pull request by daKmoR**
This is an alternative to https://github.com/webcomponents-cg/docs-and-guides/pull/2

There are primary only md or data files as the all templates are within `@rocket/lauch`.

Comes with
- Mobile Layout (including Mobile swipeable Drawer)
- Search
- Service worker that caches already visited pages so they work offline
- Social Media images
- ...

@castastrophe please take a look and then let's have a chat on how we wanna continue 🤗

Preview: https://deploy-preview-4--web-components-wg.netlify.app/

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  2. [Add a new issue](https://github.com/w3c/csswg-drafts/issues/new)
+
+Issues are where individual reports and Community Group discussions come together such 
+that the eventual consensus can be turned into official language.
+
+## Normative and/or substantive contributions
+
+The Web Components Community Group operates via the consensus of its membership, and discusses all significant matters prior to implementation.
+
+<!-- @Westbrook would this be a good place to include the working group meeting links and cadence details?
```
Comment (Westbrook) [inline CONTRIBUTING.md:27]: That sounds great. Something like:
```
The Community Group, aside from managing issues and PRs on GitHub, discusses recommendations, goals, and planning via events available on [this shared calendar](https://calendar.google.com/calendar/u/0/embed?src=o25bim5rvcu42mfnqilirpmp44@group.calendar.google.com).
```
I wish there was an easy way to leave a slack invite, but they all generally go stale, right?

Comment (daKmoR) [reply CONTRIBUTING.md:27]: > Each invite link can be used by up to 2,000 people.

I think that will work long enough 😬
Once we 2000+ people then we probably can invest in automating this further

Comment (bennypowers) [reply CONTRIBUTING.md]: Addressed this in c6dded6f1bbf7bfaf895f8acc1eb907bf5f42a8a

I wasn't able to ["edit link settings"](https://slack.com/intl/en-il/help/articles/201330256-Invite-new-members-to-your-workspace#share-an-invitation-link) to create a permanent invite link

**Review by Westbrook — APPROVED** [review]
LGTM!

Happy to have a starting place here. 🚀

**Comment by bennypowers** [comment]
x-post from slack, as of this comment we have

- :white_check_mark: site skeleton
- :white_check_mark: sample page
- :white_check_mark: dark mode css
- :white_check_mark: webmanifest and icons
Still to do
- :no_entry_sign: Initial content
- :no_entry_sign: sitemap (i.e. taxonomy and overall structure)
- :no_entry_sign: demos, codelabs, interop article
I've got plans for that interop article, theoretically I could start even without merging this, but I'd be more comfortable branching from main if possible. We can always shuffle content later then do a 'soft' launch


I'd like to get @castastrophe 's approval before we merge

**Review by bennypowers — APPROVED** [review]

**Pull request merged**